src/lib/onboarding/router.ts

```diff
@@ -97,6 +103,31 @@ router.post('/submit-verified', async (req, res) => {
   }
 });
 
+/// FALLBACK ONLY
+router.post('/submit-with-sig', async (req, res) => {
+  try {
+    const accountWithSig = req.body;
+
+    if (!validateAccountWithSig(accountWithSig))
+      throw new AccountWithSigNonValidError(accountWithSig);
+
+    accountWithSig.email = accountWithSig.email.toLowerCase();
+    accountWithSig.address = accountWithSig.address.toLowerCase();
+
+    await OnBoardingService.getInstance().submitAccountWithSig(accountWithSig);
+
+    return res.status(201).send('Ok');
+  } catch (e) {
+    logger.error(req.path, e);
+    res.status(403).send({
+      error:
+        e instanceof OnBoardingError
```

Comment:
`OnBoardingError` is the grand-grand-paranet of all errrors? Then good.

Reply:
yap of all